serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/controller/ServiceRecordController.java

```diff
@@ -114,7 +121,13 @@ public ResponseEntity entity(
         }
 
         if (!serviceRecord.isPresent()) {
-            serviceRecord = serviceRecordResponseHandler(serviceRecordFactory.createFiksServiceRecord(identifier), entity);
+            Optional<FiksWrapper> fiksWrapper = serviceRecordFactory.createFiksServiceRecord(identifier);
+            if (fiksWrapper.isPresent()) {
+                serviceRecord = serviceRecordResponseHandler(Optional.of(fiksWrapper.get().getServiceRecord()), entity);
+                if (serviceRecord.isPresent()) {
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Change this condition so that it does not always evaluate to "false" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS2583)